jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SkippedGoalVerifier.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 Google LLC. All rights reserved.
```

Comment:
I meant to ask about this _All rights reserved._ It's [not normally seen on ASL headers](https://www.apache.org/licenses/LICENSE-2.0.html#apply), and @elharo indicated that there was [some internal discussion about whether to include it or not](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/2964#issuecomment-375420457).

Reply:
Okay, yeah I just looked at the updating docs, and `All rights reserved` is gone. I'll update that, and then push a larger update across the codebase.